README.md

```diff
@@ -83,7 +83,8 @@ There are many .NET related projects on GitHub.
 
 ### Daily Builds
 
-Daily builds of .NET Core components are published to [dotnet-core MyGet gallery](https://dotnet.myget.org/gallery/dotnet-core).
-The latest version number of each library can be seen in that gallery.
+Daily builds of .NET Core components are published to dotnet-blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
+The latest version number of each library can be seen in that feed.
+Currently, there is no website to visualize the contents of the feed, so in order to do so, you have to use a NuGet feed explorer, like Visual Studio.
```

Comment:
Can I do it with nuget.exe somehow? This doesn't work with latest nuget.exe:
```
C:\>\t\nuget list -Prerelease -Source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
WARNING: This version of nuget.exe does not support listing packages from package source 'https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json'.
No packages found.
```

Reply:
cc: @jcagme what is the story for this?